.github/scripts/check_diff.py

```diff
@@ -31,8 +31,8 @@
             )
         ):
             # add all LANGCHAIN_DIRS for infra changes
-            dirs_to_run["extended-test"].update(NVIDIA_DIRS)
-            dirs_to_run["lint"].add(".")
+            # dirs_to_run["lint"].add(".")
```

Comment:
Not sure about this one, but it looks like it's at most a benign issue. I will be adding docs here actually (or maybe I need to work out that workflow with LC, since we do need to update docs)

Reply:
let's revisit when you add docs. right now the root of the repo is not setup for linting. you can see the result in the previous pipeline run - https://github.com/langchain-ai/langchain-nvidia/actions/runs/8206639341

Reply:
Gotcha! SGTM!